queries.sql

```diff
@@ -36,3 +36,24 @@ SELECT AVG (weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT MAX(weight_kg), MIN(weight_kg) FROM animals;
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+
+-- make queries using join
+SELECT animals.name FROM animals
+INNER JOIN owners ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT * FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'pokemon';
+
+SELECT animals.name, owners.full_name FROM ANIMALS
+RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+SELECT species.name, COUNT(animals.species_id) FROM animals
+JOIN species ON species.id = animals.species_id GROUP BY species.name;
+
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell' AND animals.species_id = 2; 
+
```

Comment:
- [ ] Instead of adding a `WHERE` clause for the `species_id` to equal `2`, kindly join the `species` table with the rest of the tables and use the name of the species instead of its `id`. As per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=List%20all%20Digimon%20owned%20by%20Jennifer%20Orwell.) requirement, you should use the name of the species and not its `id`.